app.js

```diff
@@ -9,3 +9,11 @@ function subtract(x, y) {
 function multiply(x, y) {
   return x * y;
 }
+
+function divide(x, y) {
```

Comment:
hey these variables names aren't that descriptive. can we replace them throughout with something like
```suggestion
function divide(num1, num2) {
```